frontend/packages/dev-console/src/components/pipelines/PipelinesPage.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { FireMan_ as FireMan } from '@console/internal/components/factory';
 import { Firehose } from '@console/internal/components/utils';
-import { DevPreviewBadge } from '@console/shared';
+import { getBadgeFromType } from '@console/shared';
```

Comment:
@christianvogt I couldn't find an easy way to make use of this in lower classes. Both use-cases here are generic and don't have access to the kindObj.